packages/big-design/src/components/ButtonGroup/spec.tsx

```diff
@@ -10,14 +10,17 @@ beforeAll(() => {
   Object.defineProperties(window.HTMLElement.prototype, {
     offsetWidth: {
       get() {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```

Comment:
You can fix/remove all these by providing typings to `this`

```
      get(this: HTMLElement) {
```

(or whatever other element you're using)